aw_watcher_terminal/config.py

```diff
@@ -17,6 +17,7 @@ def load_config() -> None:
     global logger
     global verbose
     global testing
+    global is_running
```

Comment:
It's really confusing to have a file named config.py which defines lots of global variables, either you name it globals.py or preferably move it to main.py

Reply:
Yes, I will definitely rename it and consider moving some to a local place if possible.

Do you think globals.py is safe to use? It seems like the most declarative name, but when doing `import globals` it could interfere with the globals keyword...? I guess `shared.py` or `global_vars.py` seems better suited.

Reply:
Yeah you are correct, globals is not a good name. As i said, preferably move the declarations to main.py and try to pass as much as possible instead.

Reply:
Yes, I will definitely rename it and consider moving some to a local place if possible.

Do you think globals.py is safe to use? It seems like the most declarative name, but when doing `import globals` it could interfere with the globals keyword...? I guess `shared.py` or `global_vars.py` seems better suited.